new/components/mgmtworker/scripts/create.sh

```diff
@@ -52,4 +81,12 @@ if [ ! -z ${REST_SERVICE_SOURCE_URL} ]; then
     install_module "/tmp/workflows" ${MGMTWORKER_VIRTUALENV_DIR}
 fi
 
+extract_github_archive_to_tmp ${manager_repo}
+ctx logger info "Configuring Management worker..."
+deploy_blueprint_resource "${CONFIG_REL_PATH}/worker_conf.py" "${CELERY_WORK_DIR}/worker_conf.py"
+replace "(( broker_port ))" "${BROKER_PORT}" "${CELERY_WORK_DIR}/worker_conf.py"
```

Comment:
We do, it was just added after I did the initial mods on this. I've noted that I need to go through and runtime-attribute these so that they can be rendered in the same way as the rest.